### PR TITLE
About page

### PR DESCRIPTION
**Pull request by yuens1002**
please make sure we merge the last pull request from poem-by-subject branch to master before this merge.  this one has all the updates i made to about page.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+			  <li><h3>Connect Us</h3>
+				  <a href="#"><i class="fa fa-github fa-2x" aria-hidden="true"></i></a>
+				  <a href="#"><i class="fa fa-facebook fa-2x" aria-hidden="true"></i></a>
+				  <a href="#"><i class="fa fa-twitter fa-2x" aria-hidden="true"></i></a>
+				  <a href="#"><i class="fa fa-google-plus fa-2x" aria-hidden="true"></i></a>
+			  </li>
+			  <li></li>	
+			</ul>
+        </div>
+        <div class="col-md-8">
```
Comment (ghost) [inline about.html:104]: The Contact section might be better as full width below the Mission and Team divs. My thinking is that as a user, I expect forms to be below other content. Moving it down would also draw more attention to the Mission and Team content.

**Comment by JBigs206** [comment]
We might also want to add a message that lets the user know when the contact form is submitted. Even just a simple alert message so the the user can see something that shows that their message was submitted successfully.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+			  <div class="form-group">
+				<label>Your Name</label>
+				<input type="email" class="form-control" placeholder="your name">
+			  </div>
+			  <div class="form-group">
+				<label>Message</label>
+				<textarea class="form-control" rows="5" placeholder="your message goes here..."></textarea>
+			  </div>
+			  <button type="submit" class="btn btn-default">Send</button>
+			</form>
```
Comment (JBigs206) [inline about.html:116]: We might also want to add a message that lets the user know when the contact form is submitted. Even just a simple alert message so the the user can see something that shows that their message was submitted successfully.

Comment (ghost) [inline about.html:116]: @JBigs206 I like this idea. A 50px bar across the top with a message would suffice.

**Pull request merged**